normalization_differential_test.go

```
@@ -0,0 +1,156 @@
+			lowered = true
+		}
+		return !lowered
+	})
+	return lowered
+}
+
+// packageURLModuleDir locates the pinned packageurl-go source in the module
+// cache, so the test reads the version this module actually compiles against.
+func packageURLModuleDir(t *testing.T) string {
+	t.Helper()
+	out, err := exec.Command("go", "list", "-m", "-f", "{{.Dir}}", "github.com/package-url/packageurl-go").Output()
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)